editor/src/components/canvas/dom-walker.ts

```diff
@@ -245,6 +245,7 @@ function useInvalidateScenesWhenSelectedViewChanges(
   invalidatedSceneIDsRef: React.MutableRefObject<Set<string>>,
   invalidatedPathsForStylesheetCacheRef: React.MutableRefObject<Set<string>>,
 ): void {
+  const [, forceUpdate] = React.useReducer((c) => c + 1, 0)
```

Comment:
We use this pattern in a number of places, so it's maybe worth adding a hook specifically for it (so that it's always clear why the pattern is being used)